### PR TITLE
pretty file index and page layout

### LINKED ISSUE #1: prettier errors
make some html for the errors

### LINKED ISSUE #13: prettier file index
something to consider, make some kind of template for the file index list.. it's just a ul currently.

ideally, possibly we could do a common header that holds style sheet stuff that could be used for any of our custom pages.. and a common footer if you think it's necessary.. just a thought

### PR DESCRIPTION
**Pull request by skinnyjames**
Fixes #13 and extends rendering functionality a tad.
Also the layout looks a little better imo

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -139,9 +151,41 @@ Glance.prototype.serveRequest = function Glance$serveRequest (req, res) {
...
+}
+
+function errorTitle (errorCode) {
+  var mappings = {
+    '404': 'File Not Found',
+    '403': 'Forbidden',
+    '405': 'Method Not Allowed',
+    '500': 'Internal Server Error'
+  }
+  return mappings['' + errorCode]
```
Comment (jarofghosts) [inline index.js]: you shouldn't need the `'' + ` here

Comment (skinnyjames) [reply index.js]: Oh sure, I didn't remember/realize that toString couldn't be called on numeric literals but could on variables.  pushing a fix.

**Comment by jarofghosts** [comment]
published `glance@3.0.5`, thanks!

**Comment by skinnyjames** [comment]
no problemo.

**Pull request merged**